pennylane_lightning/src/tests/CMakeLists.txt

```diff
@@ -66,11 +64,12 @@ endif()
 ################################################################################
 
 add_executable(compile_time_tests compile_time_tests.cpp)
-target_link_libraries(compile_time_tests lightning_gates lightning_utils lightning_simulator)
+target_link_libraries(compile_time_tests lightning_compile_options lightning_gates lightning_utils lightning_simulator)
 
 set(TEST_SOURCES CreateAllWires.cpp
                  Test_AdjDiff.cpp
 #                 Test_Bindings.cpp
```

Comment:
Because `Test_Bindings.cpp` is empty now. We do not test any binding functions now... I think most of the binding functionalities are now tested on Python side, but we may add something in this file later if there are some essential functionalities to test.